activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -102,6 +103,12 @@ class PostgreSQLAdapter < AbstractAdapter
         ltree:       { name: "ltree" },
         citext:      { name: "citext" },
         point:       { name: "point" },
+        line:        { name: "line" },
+        lseg:        { name: "lseg" },
+        box:         { name: "box" },
+        path:        { name: "path" },
+        polygon:     { name: "polygon" },
+        circle:      { name: "circle" },
```

Comment:
As long as we have `valid_type?` implemented the way it is now, we should probably add those for consistency. Although my guess is that most people using these kind of data types are probably relying on `db/structure.sql` already.